src/Sarif.Viewer.VisualStudio/ErrorList/FilteredColumnValue.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Sarif.Viewer.ErrorList
+{
+    // Associates an error list column with a value that has been filtered from it.
+    internal class FilteredColumnValue
```

Comment:
>FilteredColumnValue [](start = 19, length = 19)

Just curious, attributes of this class, value type comparison, immutability after init, data only/no behavior, suggests you could simply prefer a value type. Is there a reason not to?

Reply:
No reason not to, good catch. Changed to `struct`.

---
In reply to: [538749153](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r538749153) [](ancestors = 538749153)